src/commands/run.rs

```diff
@@ -0,0 +1,263 @@
+//! Implementation of the run command.
+
+use std::fs;
+use std::path::Path;
+use std::path::PathBuf;
+use std::path::absolute;
+
+use anyhow::Context;
+use anyhow::Result;
+use anyhow::anyhow;
+use anyhow::bail;
+use chrono;
+use clap::Parser;
+use codespan_reporting::files::SimpleFile;
+use codespan_reporting::term::emit;
+use tracing_log::log;
+use url::Url;
+use wdl::analysis::path_to_uri;
+use wdl::engine::Engine;
+use wdl::engine::EvaluationError;
+use wdl::engine::Inputs;
+use wdl::engine::local::LocalTaskExecutionBackend;
+use wdl::engine::v1::TaskEvaluator;
+
+use crate::Mode;
+use crate::analyze;
+use crate::get_display_config;
+
+/// Arguments for the run command.
+#[derive(Parser, Debug)]
+#[command(author, version, about)]
+pub struct RunArgs {
+    /// The path or URL to the WDL document containing the task to run.
```

Comment:
Do we want to write these as `task`? Eventually, it will support workflows, so should the comments reflect that or just the current functionality?

Reply:
I think we should reflect the current functionality, i.e. specify that we only look at tasks for now. I wouldn't be opposed to making the language more agnostic as a future-proof, so long as it doesn't imply anything we don't currently support.